### ISSUE TITLE
Make images available in the AWS ECR Public Gallery

### ISSUE THREAD

**Issue by tomcrawf90** [lambci/docker-lambda#326]
Would it be possible to get this image made available on the [AWS ECR Public Gallery](https://gallery.ecr.aws/)? This would help avoid the issue of rate limiting implemented by DockerHub especially when running in CI/CD and on shared infrastructure.

**Comment by zoellner** [comment]
Came here to ask for the same. Given the timing it looks to me like Docker started to restrict AWS CodeBuild/CodePipeline IPs more now

**Comment by laynr** [comment]
> Came here to ask for the same. Given the timing it looks to me like Docker started to restrict AWS CodeBuild/CodePipeline IPs more now
> 
> edit: in the meanwhile, I created a temporary registry to host the image I need. let me know if anyone wants me to push another image. https://gallery.ecr.aws/n8r6f1x4/lambci-temporary

@zoellner please add python 3.8

**Comment by zoellner** [comment]
Added 
```
public.ecr.aws/n8r6f1x4/lambci-temporary:python3.8
sha256:bbdb78040d8067bac553dfa1b5e8b2f14e9771980b39a3325c8323b302bf1ea5
```
and 
```
public.ecr.aws/n8r6f1x4/lambci-temporary:build-python3.8
sha256:d944b5ae251d24a089c4cc8c889e861cca6ce0ea0da376c364eeebe9ea4cce58
```

Depending on what you want to do that AWS image might work fine. In my experience they're good to run stuff but if you want to build things it doesn't work well since they've stripped it down too much for that. My experience is from nodejs only however, might be different in the python world.

**Comment by kristianpaul** [comment]
Any plans to add some older python version images as well?

**Comment by zoellner** [comment]
no plans to add anything from my side since I got what I need. but if you have a particular image you need, let me know and I'll add it

**Comment by kristianpaul** [comment]
> no plans to add anything from my side since I got what I need. but if you have a particular image you need, let me know and I'll add it

lambci/lambda:build-python3.6

**Comment by zoellner** [comment]
Added
```
public.ecr.aws/n8r6f1x4/lambci-temporary:build-python3.6
sha256:a977ca47a1ee77735c83ca3d994293dc431c3655daca42ce1e9ba546963c7d8c
```

**Comment by rexm** [comment]
Can you please add `lambci/lambda:java8`?

**Comment by zoellner** [comment]
Added
```
public.ecr.aws/n8r6f1x4/lambci-temporary:java8
sha256:46f0df6293fd8b92274b7b0081bcafadda4dfb4b9bf29196f13b88415c0bb1d7
```

**Comment by rexm** [comment]
A bit embarrassing, but can you please also add:
- `lambci/lambda:nodejs6.10`
- `lambci/lambda:nodejs8.10`
- `lambci/lambda:python3.7`

**Comment by zoellner** [comment]
Added
```
public.ecr.aws/n8r6f1x4/lambci-temporary:nodejs6.10
sha256:27aba2358e8cace3a830f29bef1f7e8d10a2cfff151712b166c19bd7b86a8a82
```

```
public.ecr.aws/n8r6f1x4/lambci-temporary:nodejs8.10
sha256:f3a37b9fc2afcdf63dd8741c48d9b3f9ed0892eed8c490dd50ac7dcc75f7aee3
```

```
public.ecr.aws/n8r6f1x4/lambci-temporary:python3.7
sha256:98aa6d05a076ed08395e2ad62f119df6a9362aa0f59ac616396f924cee3f59f6
```

@rexm I hope you're not using those node versions in public anymore 🙈

**Comment by rexm** [comment]
It’s in some old local dev stuff we are upgrading :) thanks!

**Comment by rexm** [comment]
Can you please add `lambci/lambda:go1.x`?

**Comment by zoellner** [comment]
here you go:
```
public.ecr.aws/n8r6f1x4/lambci-temporary:go1.x
```

**Comment by zacthompson** [comment]
🥺 `lambci/lambda:java11` ?

**Comment by zoellner** [comment]
I added that at `public.ecr.aws/n8r6f1x4/lambci-temporary:java11` (forgot to mention it here)

**Comment by zacthompson** [comment]
> I added that at `public.ecr.aws/n8r6f1x4/lambci-temporary:java11` (forgot to mention it here)

Yes, I saw, could only react to my own comment at the time. Thank you kindly

**Comment by ohalay** [comment]
@zoellner 
Can you please add `lambci/lambda:dotnetcore3.1`?

**Comment by zoellner** [comment]
Added
```
public.ecr.aws/n8r6f1x4/lambci-temporary:dotnetcore3.1
sha256:4ee19e6a9647fb744c3cd9c9701c1bedc6a9bf3b47e5a983766819cc92bbb829
```

**Comment by rexm** [comment]
@zoellner Hey there! Can you please mirror `nodejs12.x`?

**Comment by zoellner** [comment]
```
public.ecr.aws/n8r6f1x4/lambci-temporary:nodejs12.x
digest: sha256:098709a2d12098c2ab5ad45138a2f97d3acc1788c2855b3659dd20eed62fd2af
```
